glue/viewers/image/state.py

```diff
@@ -450,10 +507,12 @@ def slice_to_bound(slc, size):
 
         if isinstance(self.layer, BaseData):
             image = self.layer.compute_fixed_resolution_buffer(full_view, target_data=self.viewer_state.reference_data,
-                                                               target_cid=self.attribute, broadcast=False, cache_id=self.uuid)
+                                                               target_cid=self.attribute, broadcast=False, cache_id=self.uuid,
+                                                               affine_transform=self.viewer_state._affine_pretransform.inverse)
```

Comment:
This is raising on the initial display when `_affine_pretransform` is still undefined (`NoneType`)

Reply:
Oops yes indeed